db/migrate/20161109150714_add_templates_to_resolve_url.rb

```diff
@@ -0,0 +1,21 @@
+class AddTemplatesToResolveUrl < ActiveRecord::Migration[5.0]
+  def up
+    Agents::WebsiteAgent.find_each do |agent|
+      keys = agent.event_keys
+      next if keys.nil? || !keys.include?('url') || agent.options.key?('template')
+      agent.options['template'] = keys.each_with_object({}) { |key, template|
+        case key
+        when 'url'
+          template[key] = '{{ url | to_uri: _response_.url }}'
+        else
+          template[key] = "{{ #{key} }}"
```

Comment:
Ah, I missed that. I don't know-- both make sense to me. I suppose there could be an option to switch between, but that might be excessive. If I had to pick one, I might do merge instead of replace (what we have now), but since it's been released, I don't think it's worth changing.